### ISSUE TITLE
Selecting an option from a dropdown box does not take

### ISSUE THREAD

**Issue by BrianTullio** [pafcu/ConfigObj-GUI#6]
When you select an option from a dropdown box, it doesn't trigger the usual routines. The GUI accepts it but the config does not change or get saved. If you hit the restore defaults button on the dropdown, it triggers some routines but still does not update the config.

**Comment by pafcu** [comment]
I can't seem to reproduce this? Can you provide a minimal example showning the problem?

**Comment by BrianTullio** [comment]
Sorry, I think I created this issue prematurely.

I recently inherited a project and it looks like the previous developer got most of the way through converting ConfigObj-GUI to PyQt5.

Since the ComboBox widget passes the handler function as a string, it is handled differently than the other widgets. 

PyQt5 doesn't like this line:
`QtCore.QObject.connect(self.main_widget, QTCore.SIGNAL(signal), func)`

Seems I just need to find a way to connect 'currentIndexChanged(QString)' under PyQt5

**Comment by pafcu** [comment]
Someone actually uses this? I actually did not really expect that! I haven't even really looked at the code for the past 7 years. I e.g. noticed that it does not actually work with Python 3 (maybe fixed now).

**Comment by BrianTullio** [comment]
Yep. And besides this ComboBox thing, it seems to work and do what we need it to.

We are trying to synchronize settings between a GUI host and a non-GUI client app. This really helps since we don't have to worry about updating the GUI if we remove or add a setting.

**Comment by pafcu** [comment]
Would you mind publishing your work on PyQt5, even if it is partial? I could perhaps upstream it and take a look at the ComboBox issue

**Comment by BrianTullio** [comment]
Sure. Although, I am a Git/GitHub novice and don't really know how to do that. :-/

**Comment by pafcu** [comment]
Depends a bit on how you got the code, and if you still have it in a git repository. If it feels complicated, you can just put it in a .zip file on Google drive or similar, and I'll grab it from there.

**Comment by pisaac-ozflux** [comment]
Well. @pafcu , I guess it never rains but it pours ...
I was intrigued when I saw @BrianTullio post here just hours after I found this repo.  I, too, have just discovered ConfigObj-GUI and am excited because it would seem that, with a little modification, it will do just what I want.  I, too, have tried to port to Py3 and PyQt5 but this was a bit ambitious given that I have never used Qt ...  Anyway, reverted to Py27 and PyQt4 and it all seems to work.
As a little background, I have written a Python program for processing data collected by sites measuring the exchange of H2O and CO2 between the surface of the Earth and the atmosphere (see https://github.com/OzFlux/PyFluxPro).  The sites I use are in Australia (see http://www.ozflux.org.au/) but the program works just as well for global sites (see http://fluxnet.fluxdata.org/).  The program uses ConfigObj files to control what it does and to date users have had to edit the ConfigObj files with a text editor.  Not very friendly.  I've just started converting the simple Tkinter GUI that runs it to Qt and hope to use ConfigObj-GUI (with acknowledgement of course) to provide a way for users to edit the ConfigObj files within the application.
So, many thanks @pafcu for your effort in producing what looks like a very useful module!
